app/Controllers/CommentControllers.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controllers;
+
+use App\View;
+use App\Models\Post;
+use App\Models\User;
+use App\Models\Comment;
+
+class CommentControllers
+{
+
+    protected $commentModel;
+
+    public function __construct()
+    {
+        $this->postModel = new Post();
```

Comment:
Undefined properties 'postModel' and 'userModel' in the file, need to be declared the same way as 'commentModel''s

Reply:
Heya Derrick, 
Im not sure what you mean by this? Are you able to clarify with me?

Reply:
You declared `$commentModel` on line 15, but not `$postModel` and `$userModel`

Reply:
roger roger